test/compare-acorn-es2020/optional-chaining.expected.js

```diff
@@ -0,0 +1,47 @@
+obj.aaa.bbb;
+obj.aaa?.bbb;
+obj?.aaa.bbb;
+obj?.aaa?.bbb;
+obj.aaa.bbb;
+obj.aaa?.bbb;
+(obj?.aaa).bbb;
+(obj?.aaa)?.bbb;
+(obj?.aaa).bbb.ccc.ddd;
+((obj?.aaa).bbb?.ccc).ddd;
+(obj?.aaa)?.bbb;
+obj[aaa][bbb];
+obj[aaa]?.[bbb];
+obj?.[aaa][bbb];
+obj?.[aaa]?.[bbb];
+obj[aaa][bbb];
+obj[aaa]?.[bbb];
+(obj?.[aaa])[bbb];
+(obj?.[aaa])?.[bbb];
+obj[aaa][bbb][ccc][ddd];
+((obj?.[aaa])[bbb]?.[ccc])[ddd];
+1?.a;
+obj()();
+obj()?.();
+obj?.()();
+obj?.()?.();
+obj()();
+obj()?.();
+(obj?.())();
+(obj?.())?.();
+obj()()()();
+((obj?.())()?.())();
+(a?.b)();
+a?.b();
+a?.b?.();
+(a?.b)?.();
+a?.().b;
+(a?.()).b;
+a?.b.c();
+(a?.b.c)();
+a.b?.().c;
+(a.b?.()).c;
+(a.b?.())?.c;
+new (a?.b().c)();
+new (a?.b())();
```

Comment:
The trailing parens on these two lines are unnecessary. I think we're missing some special handling in the generation of `NewExpression`.